src/components/DatePicker/DatePicker.jsx

```diff
@@ -205,7 +180,7 @@ export const DatePicker = ({
             {components.Footer ? <components.Footer /> : null}
 
             {useDateRangeStyle && shouldShowRelativeRanges && (
-                <div className="ml-auto w-6/12 pl-5 pr-10 pb-5">
+                <div className="ui-range-picker-wrapper flex flex-row-reverse">
```

Comment:
* `ui-range-picker-wrapper` needs to match the width of the date picker whose date is set in css which is why I added this custom class
* `flex-row-reverse` basically solves everything